index.html

```diff
@@ -2470,15 +2466,30 @@ <h2>Production and Consumption</h2>
 
       <ol>
         <li>
-A <a>conforming consumer</a> MUST add all entries that do not have explicit
-processing rules for the <a>representation</a> being consumed to the <a
-href="#data-model">data model</a> using only the <a>representation</a>'s data
-type processing rules.
+A <a>conforming consumer</a> MUST take a <a>representation</a> and
+Media Type <a data-cite="INFRA#string">string</a> as input into
+the <a>consumption</a> process. A <a>conforming consumer</a> MAY accept
+additional options as input into the <a>consumption</a> process.
         </li>
         <li>
 A <a>conforming consumer</a> MUST determine the <a>representation</a> of a
-<a>DID document</a> using the associated Media Type as described in <a
-href="#did-resolution-metadata"></a>.
+<a>DID document</a> using the Media Type input <a
+data-cite="INFRA#string">string</a>.
+        </li>
+        <li>
+A <a>conforming consumer</a> MUST detect any representation-specific
+entry across all known <a>representations</a> and place the entry into a
```

Comment:
Perhaps we want to say "MUST detect any representation-specific entry known to the representation being consumed", and "SHOULD/MAY detect representation-specific entries across all other known representations"? Anyway I'm also fine with this as-is.

Reply:
Yes, I had considered that and decided not to go that route because it makes detection of other representation-specific entries optional, which would ultimately lead to interoperability issues on a critical feature needed to solve the problem we're trying to solve. Making this behavior mandatory removes the possibility of non-interoperability at the representation-specific entries data model.